Shops/ShopClothier.cs

```diff
@@ -78,69 +78,51 @@ public void InitShop(string currentShop)
                     if (Main.bloodMoon)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.CenxsDressPants);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.CenxsLeggings);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.CenxsDress);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.CenxsBreastplate);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.CenxsTiara);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
                     }
 
                     if (Main.eclipse)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.JimsLeggings);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.JimsBreastplate);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
 
                         shop.item[nextSlot].SetDefaults(ItemID.JimsHelmet);
-                        shop.item[nextSlot++].shopCustomPrice = 100000;
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.